tests/topology/expected/6pe.yml

```diff
@@ -135,6 +135,9 @@ nodes:
       advertise_loopback: true
       as: 65001
       community:
+        confed_ebgp:
```

Comment:
Could limit the community types to those being used in the topology, and avoid all these changes

Reply:
In theory, someone might want to limit the spread of extended communities to member autonomous systems. Also, the changes to test results don't hurt if we know why they happen.

Reply:
... however, it's trivial to enable "confed_ebgp" value only when the topology uses confederations, so why not 😎